src/clzones.cpp

```diff
@@ -582,16 +648,52 @@ bool zone_manager::has_loot_dest_near( const tripoint &where ) const
     return false;
 }
 
+const zone_data *zone_manager::get_zone_at( const tripoint &where ) const
```

Comment:
Since zones can overlap¹, should there be a version of this method that can take a `zone_type_id` to make sure it's returning the type of zone that's being asked for?

—

¹ The classic example being overlaying an 'unsorted' over your food supplies, so empty containers get moved out.

Reply:
That function should be like that anyway probably, otherwise it wont distinguish between layered zones. ill do that.